module/Kanbanize/src/Kanbanize/Service/ImportDirector.php

```diff
@@ -21,7 +21,7 @@
 class ImportDirector implements EventManagerAwareInterface{
 	
 	CONST IMPORT_COMPLETED = "Import.Completed";
```

Comment:
Attribuisci una stringa più specifica. `Import.Completed` potrebbe in futuro confliggere con altri processi di import. Adotta un `KanbanizeImport.Completed` per esempio